frontend/src/components/eMIB/EmailPreview.jsx

```diff
@@ -1,30 +1,115 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
 
 const styles = {
+  //buttons
   button: {
-    width: 180,
-    marginBottom: 10,
-    marginLeft: 20,
-    marginRight: 20,
-    textAlign: "center",
+    width: 155,
+    textAlign: "left",
     borderRadius: 4,
     padding: 6,
-    border: "2px solid #009fae"
+    border: "2px solid #00565E"
+  },
+  // 3x background colors
+  button_selected_background: {
+    backgroundColor: "#00565E"
+  },
+  button_read_background: {
+    backgroundColor: "#F5F5F5"
```

Comment:
You should also put _cursor: "pointer"_, so the cursor would be different when the mouse is over the read email box.

Reply:
fixed in a518cf4